src/vmm/src/builder.rs

```diff
@@ -833,7 +840,13 @@ pub fn configure_system_for_boot(
                     vmm.guest_memory(),
                     entry_addr,
                     &vcpu_config,
-                    vmm.vm.supported_cpuid().clone(),
+                    X86_64CpuConfiguration {
+                        cpuid: Cpuid::try_from(
+                            RawCpuid::from(vmm.vm.supported_cpuid().clone()).clone(),
```

Comment:
Does this need to do `clone()` multiple times?
```suggestion
                            RawCpuid::from(vmm.vm.supported_cpuid().clone()),
```